web/middleware/urlquery.go

```diff
@@ -0,0 +1,24 @@
+package middleware
+
+import (
+	"github.com/zenazn/goji/web"
+	"net/http"
+)
+
+// UrlQueryKey is the context key for the URL Query
+const UrlQueryKey string = "urlquery"
```

Comment:
URL should be capitalized here and elsewhere (e.g., https://golang.org/pkg/net/http/#ProxyURL)